src/api-service/__app__/onefuzzlib/workers/nodes.py

```diff
@@ -133,7 +133,7 @@ def search_outdated(
     @classmethod
     def mark_outdated_nodes(cls) -> None:
         # if outdated agents are allowed, do not attempt to update
-        if os.environ["ONEFUZZ_ALLOW_OUTDATED_AGENT"] == "true":
+        if os.environ.get("ONEFUZZ_ALLOW_OUTDATED_AGENT") == "true":
```

Comment:
don't forget to port the logic to the C# code

Reply:
The C# version already does the right thing here.